cmake/CPM.cmake

```diff
@@ -5,7 +5,7 @@
 # MIT License
 # -----------
 #[[
-  Copyright (c) 2021 Lars Melchior and additional contributors
+  Copyright (c) 2021 Lars Melchior, NVIDIA CORPORATION, and additional contributors
```

Comment:
I didn't see a place that listed contributors, so I added NVIDIA here.
If you don't want I can drop